nixos/tests/osquery.nix

```diff
@@ -0,0 +1,61 @@
+import ./make-test-python.nix ({ lib, pkgs, ... }:
+
+with lib;
+
+let
+  config_refresh = "10";
+  logger_path = "/var/other/log/path";
+  nullvalue = "NULL";
+  utc = false;
+in
+{
+  name = "osquery";
+  meta = with maintainers; {
+    maintainers = [ jdbaldry znewman01 ];
+  };
+
+  nodes.machine = { config, pkgs, ... }: {
+    services.osquery = {
+      enable = true;
+
+      settings.options = { inherit nullvalue utc; };
+      flags = {
+        inherit config_refresh logger_path;
+        nullvalue = "IGNORED";
+      };
+    };
+  };
+
+  testScript = { nodes, ... }:
+    let
+      cfg = nodes.machine.config.services.osquery;
+      flags = import ../modules/services/monitoring/osquery/flags.nix { inherit cfg lib pkgs; };
```

Comment:
are users supposed to need this information? if so, I think it would be relevant to either:

- put it in `/etc/osquery/flags.conf` or something like this as a symlink
- wrap `osqueryi` to provide it automatically akin to what we do with other services

Reply:
I don't think users need this information. 
At least, i'm able to use the osqueryi binary without any additional configurations or cli options.